Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -853,7 +855,9 @@ - (void)testSpecTests {
         NSLog(@"  Spec test: %@", name);
         [self runSpecTestSteps:steps config:config];
         ranAtLeastOneTest = YES;
+        testPassCount++;
```

Comment:
Google style guide suggest:

Use the prefix form (++i) of the increment and decrement operators unless you need postfix semantics.

https://engdoc.corp.google.com/eng/doc/devguide/cpp/styleguide.md?cl=head#Preincrement_and_Predecrement

Reply:
Done.  Changed to the prefix operator.